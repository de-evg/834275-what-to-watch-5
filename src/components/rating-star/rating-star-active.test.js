import React from "react";
import renderer from "react-test-renderer";
import RatingStar from "./rating-star";


it(`Should RatingStar active render correctly`, () => {
  const tree = renderer
    .create(<RatingStar
      starId={`star-0`}
      starValue={1}
      isActive={true}
      onRatingInputChange={() => {}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
