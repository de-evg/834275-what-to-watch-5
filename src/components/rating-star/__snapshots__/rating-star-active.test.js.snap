// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should RatingStar active render correctly 1`] = `
Array [
  <input
    checked={true}
    className="rating__input"
    id="star-0"
    name="rating"
    onChange={[Function]}
    required={true}
    type="radio"
    value={1}
  />,
  <label
    className="rating__label"
    htmlFor="star-0"
  >
    Rating 
    1
  </label>,
]
`;
